apps/vaults/contexts/useSolver.tsx

```diff
@@ -75,7 +74,7 @@ function WithSolverContextApp({children}: { children: React.ReactElement }): Rea
 	const partnerContract = useSolverPartnerContract();
 	const internalMigration = useSolverInternalMigration();
 	const optimismBooster = useSolverOptimismBooster();
-	const [currentSolverState, set_currentSolverState] = useState<TSolverContext & { hash: MaybeString }>({...vanilla, hash: undefined});
+	const [currentSolverState, set_currentSolverState] = useState<TSolverContext & { hash?: string }>({...vanilla, hash: undefined});
```

Comment:
nit: Since `hash` will be `undefined` if not defined, we can just do
```suggestion
	const [currentSolverState, set_currentSolverState] = useState<TSolverContext & { hash?: string }>(vanilla);
```